manager/orchestrator/taskreaper/task_reaper_test.go

```diff
@@ -897,23 +898,6 @@ func TestTick(t *testing.T) {
 	assert.NotNil(t, s)
 	defer s.Close()
 
-	assert.NoError(t, s.Update(func(tx store.Tx) error {
-		store.CreateCluster(tx, &api.Cluster{
-			ID: identity.NewID(),
-			Spec: api.ClusterSpec{
-				Annotations: api.Annotations{
-					Name: store.DefaultClusterName,
-				},
-				Orchestration: api.OrchestrationConfig{
-					// set TaskHistoryRetentionLimit to a negative value, so
-					// that tasks are cleaned up right away.
-					TaskHistoryRetentionLimit: 1,
-				},
-			},
-		})
-		return nil
-	}))
-
```

Comment:
why did all of this get removed?

Reply:
I don't know why it was removed, but I'd also like to point out that `1` isn't a negative number...
```
					// set TaskHistoryRetentionLimit to a negative value, so
					// that tasks are cleaned up right away.
					TaskHistoryRetentionLimit: 1,
```

Reply:
cluster object is not needed for this test

Reply:
Yep, it is definitely the default.  I was just suggesting that setting it would be more explicit/easier to read/consistent style-wise as it is in the other test.  Not important though.